share/completions/iftop.fish

```diff
@@ -0,0 +1,29 @@
+function __fish_iftop_sort_orders
+	echo -e "2s\tSort by 2s traffic average"
+	echo -e "10s\tSort by 10s traffic average (default)"
+	echo -e "40s\tSort by 40s traffic average"
+	echo -e "source\tSort by source address"
+	echo -e "destination\tSort by destination address"
+end
+
+complete -c iftop -f
+
+complete -c iftop -s h -d "Print a summary of usage"
+complete -c iftop -s n -d "Don't do hostname lookups"
+complete -c iftop -s N -d "Don't resolve port number to service names"
+complete -c iftop -s p -d "Run in promiscuous mode"
+complete -c iftop -s P -d "Turn on port display"
+complete -c iftop -s l -d "Include link-local IPv6 addresses"
+complete -c iftop -s b -d "Don't display bar graphs of traffic"
+complete -c iftop -s m -x -d "Set the upper limit for the bandwidth scale"
+complete -c iftop -s B -d "Display bandwidth rates in bytes/sec rather than bits/sec"
```

Comment:
I think this could be shortened with Show instead of Display and bytes/s + bits/s instead of /sec